### PR TITLE
Fail the GTG check

### PR DESCRIPTION
**Pull request by commuterjoy**
/cc @ironsidevsquincy secret

### CODE REVIEW

**Comment by ironsidevsquincy** [comment]
👍  I wont tell anyone

File: server/init.js [hunk 1]
```diff
@@ -51,8 +51,14 @@ startPolling();
 startBrexitLivePolling();
 startBrexitSecondaryLivePolling();
 
+// FIXME - temporary outage
+const hour = parseInt(process.env.simulated_outage) || 26;
```
Comment (leggsimon) [inline server/init.js]: Is this meant to be ` || 24` ?

Comment (commuterjoy) [inline server/init.js]: 26 just is an hour that doesn't exist, so it's switched off

**Pull request merged**